packetbeat/statsdns/queue_stat_dns.go

```diff
@@ -109,6 +112,7 @@ func (queue *QueueStatDNS) PopStatDNS() {
 			if recursive == nil {
 				continue
 			}
+			time.Sleep(5 * time.Microsecond)
```

Comment:
Please try to find another solution for wait/notify. Sleep is not a good solution as 5 micro seconds can be too much or too little.